CONTRIBUTING.md

```diff
@@ -72,3 +72,14 @@ OR
 * The module is under the Node.js foundation Github org
 OR
 * The module is identified as an important module by a Node.js Working Group
+
+#### Procedure
+
+After making sure you adhere to the above requirements, do the
+following:
+
+1. Add the module to
+   [`lib/lookup.json`](https://github.com/nodejs/citgm/blob/master/lib/lookup.json)
+1. Run `npm link`
+1. Make sure the `citgm <module>` tests pass
+1. Commit your changes and open a PR
```

Comment:
Maybe something like:

`1. Commit your changes and open a PR` -> `1. Commit your changes and open a PR. Include a brief answer to the requirements.`

What do you think @nodejs/citgm ? I think we should ask people to document that they meet the requirements, otherwise it's hard to justify having them.